packages/addons-website/scss/components/_website-typeset-style.scss

```diff
@@ -2,15 +2,32 @@
 
 .#{$prefix}--typeset-style-container {
   position: relative;
-  margin-left: -1rem;
-  margin-right: -1rem;
   padding: 0 0.5rem;
-  margin-top: 0;
+  margin-bottom: 3rem;
```

Comment:
I know your other PR is updating the existing padding/margin styles to get the spacing tokens, but since these are new, do you want to use tokens for them?